reference/ros-api.md

```diff
@@ -359,6 +359,7 @@ for more information about the value of `robot_unique_name`).
 | [/supervisor/get\_self](supervisor.md#wb_supervisor_node_get_from_def)                                                | service       | webots\_ros::supervisor\_get\_self                                   | uint8 ask<br/>---<br/>uint64 node                                                                                                                                                                                                                                 |
 | [/supervisor/get\_from\_def](supervisor.md#wb_supervisor_node_get_from_def)                                           | service       | webots\_ros::supervisor\_get\_from\_def                              | string name<br/>---<br/>uint64 node                                                                                                                                                                                                                               |
 | [/supervisor/get\_from\_id](supervisor.md#wb_supervisor_node_get_from_def)                                            | service       | webots\_ros::supervisor\_get\_from\_id                               | int32 id<br/>---<br/>uint64 node                                                                                                                                                                                                                                  |
+| [/supervisor/get\_selected](supervisor.md#wb_supervisor_node_get_from_def)                                            | service       | webots\_ros::supervisor\_get\_selected                               | uint8 ask<br/>---<br/>uint64 node                                                                                                                                                                                                                                 |
```

Comment:
It is not the same one as `wb_supervisor_node_get_from_def` has a string as input argument.
We may use the same as for `wb_supervisor_node_get_self` 
but it would look strange to call the `/supervisor/get_selected` service with a `webots_ros::supervisor_get_self` service type. Ideally we should rename the service type `webots_ros::supervisor_get_self` into `webots_ros::supervisor_get_node` (which is more generic) and use it for both service, but that would break the compatibility (maybe we can afford, but it doesn't seems good as we often break compatibility of the ros controller).